src/integration-tests/introspector/introspectTest.sh

```diff
@@ -123,7 +138,7 @@ function cleanupMajor() {
 
   rm -fr /tmp/introspect
   mkdir -p $test_home || exit 1
-
+  rm -fr ${test_home}/mii/workdir
```

Comment:
You don't need this line now because test_home is /tmp/introspect, and /tmp/introspect is removed two lines above.